### PR TITLE
BE: Skip link cards and actions when generate subscriptions contents

### PR DESCRIPTION
**Pull request by qnkhuat**
In 46 we ship with 2 new kinds of dashcards: link cards and actions, but we don't want to include these in actions (yet).

Even though the subscriptions won't include the actions or link cards without these changes, but we still should do this to make it explicit that we filtered it out.

Related https://github.com/metabase/metabase/issues/28426

- - -
This change is [Reviewable](https://reviewable.io/reviews/metabase/metabase/28519)

### CODE REVIEW

File: src/metabase/pulse.clj [hunk 1]
```diff
@@ -80,20 +80,48 @@
...
+    (let [parameters (merge-default-values (params/parameters pulse dashboard))]
+      (execute-dashboard-subscription-card pulse-creator-id dashboard dashcard (:card_id dashcard) parameters))
+
+    ;; actions
+    (virtual-card-of-type? dashcard "action")
+    nil
+
+    ;; link cards
+    (virtual-card-of-type? dashcard "link")
+    nil
```
Comment (qnkhuat) [inline src/metabase/pulse.clj:105]: We'll add support for link cards [PR](https://github.com/metabase/metabase/pull/28528), so this part is a bit explicit to use as template for the follow up PR.

File: src/metabase/pulse.clj [hunk 2]
```diff
@@ -80,20 +80,48 @@
     (compare (:row dashcard-1) (:row dashcard-2))
     (compare (:col dashcard-1) (:col dashcard-2))))
 
+(defn- virtual-card-of-type?
+  "Check if dashcard is a virtual with type `ttype`.
+
+  There are currently 3 types of virtual card: text, action, link."
+  [dashcard ttype]
+  (and (map? dashcard)
+       (= ttype (get-in dashcard [:visualization_settings :virtual_card :display]))))
```
Comment (calherries) [inline src/metabase/pulse.clj:89]: To make this simpler, I would have made this return the type of the dashcard, so the caller can do an equality or set inclusion. e.g. `(= (virtual-card-type dashcard) "action")`

**Review by calherries — APPROVED** [review]
This looks solid

File: src/metabase/pulse.clj (hunk 2 above)
Comment (qnkhuat) [reply src/metabase/pulse.clj:89]: Fair point, I have a follow-up PR that might change this fn, I will do it then.

**Pull request merged**